pettingzoo/utils/conversions.py

```diff
@@ -217,17 +223,21 @@ def close(self):
         return self.aec_env.close()
 
 
-class parallel_to_aec_wrapper(AECEnv):
+class parallel_to_aec_wrapper(AECEnv[AgentID, ObsType, Optional[ActionType]]):
     """Converts a Parallel environment into an AEC environment."""
 
-    def __init__(self, parallel_env):
+    def __init__(
+        self, parallel_env: ParallelEnv[AgentID, ObsType, Optional[ActionType]]
+    ):
         self.env = parallel_env
 
         self.metadata = {**parallel_env.metadata}
         self.metadata["is_parallelizable"] = True
 
         try:
-            self.render_mode = self.env.render_mode
+            self.render_mode = (
+                self.env.render_mode  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
I don't see why your code should make this specific line need a different pyright ignore, I guess it was just throwing that error now for some reason? Kind of ugly but doesn't matter much if it's not possible to remove

Reply:
Render mode is not actually defined as a property or variable on either `ParallelEnv` or `AECEnv`.

Reply:
That's a little weird, but I don't think we should change it. I imagine there are some checks which say if hasattr(render_mode) or something similar. Other subsequent training libraries assume things are how they are so I don't want to change much without a really strong reason to do so.